### ISSUE TITLE
Icon: requires a `body`. When serving content through AJAX we do not have a body.

### ISSUE THREAD

**Issue by CestDiego** [eBay/ebayui-core#206]
# Bug Report

## Description
Create a route in ajax that just does this:
```
<lasso-page
        package-path="./browser.json"
    />
 <ebay-dialog > Hello </ebay-dialog>
```
in the controller, send down json to the browser so the user can inject it.
```
        template.render({}, (err, output) => {
            const renderedComponents = require('marko/components').getRenderedComponents(output.out);
            return resolve({
                renderedComponents,
                html: output.getOutput()
            });
        });
```

Observe how there is no ebay-icon transform done in the response.

**Comment by ianmcburnie** [comment]
@DylanPiercey Feel free to close if you feel this is invalid or by design.

**Comment by ianmcburnie** [comment]
Fix will be available in 0.6.0.
